traefik/oam/register.go

```diff
@@ -117,3 +125,93 @@ func load(basePath string) ([]schemaDefinitionPathSet, error) {
 
 	return res, nil
 }
+
+//RegisterWorkLoadsDynamically ...
+func RegisterWorkLoadsDynamically(runtime, host string) error {
+	av, cv, err := getLatestValidAppVersionAndChartVersion()
+	if err != nil {
+		return err
+	}
+	fmt.Println("version ", av)
+	m := manifests.Config{
+		Name:        "Traefik",
```

Comment:
@DelusionalOptimist are we using definitions of service mesh names from SMP protos here?

Reply:
will use smp proto here in next commit. Has to add the same in all other adaptors as well.

Reply:
Thank you, sir.